rhino/src/test/java/org/mozilla/javascript/tests/Utils.java

```diff
@@ -106,21 +84,4 @@ public static void assertWithAllOptimizationLevelsES6(
                     return null;
                 });
     }
-
-    public static void assertEvaluatorExceptionES6(String expectedMessage, String js) {
-        Utils.runWithAllOptimizationLevels(
-                cx -> {
-                    cx.setLanguageVersion(Context.VERSION_ES6);
-                    ScriptableObject scope = cx.initStandardObjects();
-
-                    try {
-                        cx.evaluateString(scope, js, "test", 1, null);
-                        fail("EvaluatorException expected");
-                    } catch (EvaluatorException e) {
-                        assertEquals(expectedMessage, e.getMessage());
-                    }
-
-                    return null;
-                });
-    }
```

Comment:
Why this is removed?

Reply:
This function, and "getTestOptLevels", is no longer used. There's also a duplicate-ish version of this class in the "tests" module where this might be used more.